docs/readmes/orc8r/deploy_using_juju.md

```diff
@@ -37,13 +38,23 @@ a Juju controller on it.
 
 ## Deploy charmed Magma Orchestrator
 
-From your Ubuntu machine, create an `overlay.yaml` file that contains the following content:
+From your Ubuntu machine, create an `overlay.yaml` file that contains
+the following content:
 
 ```yaml
 applications:
+  fluentd:
+    options:
+    domain: <your domain name>
+    elasticsearch-url: <your elasticsearch https url>
+    fluentd-chunk-limit-size: "2M"
```

Comment:
Do we need to specify `fluentd-chunk-limit-size` and `fluentd-queue-limit-length` since we're using the default values?

Reply:
My point was to make user aware of these options.
Do you think something like `fluentd-chunk-limit-size: <chunk size limit, defaults to "2M">` would be better?

Reply:
I'd just keep the options that are necessary, here we don't want to cover all use cases. In other words I'd just remove those 2 lines.